apps/core/lib/core/seed/country_state_seed.ex

```diff
@@ -1,12 +1,12 @@
-defmodule Core.Seed.CountryStateSeeder do
+defmodule Snitch.Core.Seed.CountryStateSeeder do
```

Comment:
Why does this namespace have `Seed` twice? IMO `Snitch.Core.Seed.CountryState` is better.